### PR TITLE
Remove single letter beta badges

### PR DESCRIPTION
**Pull request by BSFishy**
### Description
Remove single letter beta badges, as mentioned here: https://github.com/opensearch-project/oui/issues/93#issuecomment-1306592845.

**Note:** This is a breaking change

### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/components/badge/beta_badge/_beta_badge.scss [hunk 1]
```diff
@@ -60,16 +60,6 @@
...
-.ouiBetaBadge--singleLetter {
-  padding: 0 0 0 1px;
-  width: $ouiSizeL;
-
-  &.ouiBetaBadge--small {
-    width: $ouiSize + $ouiSizeXS;
-    padding: 0 0 0 1px;
-  }
-}
-
```
Comment (joshuarrrr) [inline src/components/badge/beta_badge/_beta_badge.scss:72]: Should we note somewhere that we're not actually preventing a user from providing a single character label - we've just removed custom styling for that scenario and encouragement of that usage pattern from the doc. But have we validated the look/behavior of a single letter label without these styles? Or do we actually want to add some minimum length enforcement within the component (that prevents rendering?)

**Review by joshuarrrr — COMMENTED** [review]
Because we don't actually prevent the previous usage, I think there's an argument to be made that it's not really breaking (it didn't depend on a prop we're removing). But if it is still considered breaking, I'd suggest creating a label that marks it as such - it will make your future backporting/releasing much less painful.

**Comment by KrooshalUX** [comment]
I thought we were going to make these changes in experimental badge instead of beta badge after precious discussions. Matt and I connected today on not making this breaking by leaving beta badge as is, duplicating it and renaming it experimental and then making the changes to OUIExperimental.



On Dec 8, 2022, at 4:16 PM, Josh Romero ***@***.***> wrote:

﻿

@joshuarrrr commented on this pull request.

Because we don't actually prevent the previous usage, I think there's an argument to be made that it's not really breaking (it didn't depend on a prop we're removing). But if it is still considered breaking, I'd suggest creating a label that marks it as such - it will make your future backporting/releasing much less painful.

________________________________

In src/components/badge/beta_badge/_beta_badge.scss<https://github.com/opensearch-project/oui/pull/116#discussion_r1043944310>:

> -.ouiBetaBadge--singleLetter {
-  padding: 0 0 0 1px;
-  width: $ouiSizeL;
-
-  &.ouiBetaBadge--small {
-    width: $ouiSize + $ouiSizeXS;
-    padding: 0 0 0 1px;
-  }
-}
-


Should we note somewhere that we're not actually preventing a user from providing a single character label - we've just removed custom styling for that scenario and encouragement of that usage pattern from the doc. But have we validated the look/behavior of a single letter label without these styles? Or do we actually want to add some minimum length enforcement within the component (that prevents rendering?)

**Comment by BSFishy** [comment]
Close to redo work to align with #161

**Pull request closed without merging**